dataset/dataset.go

```diff
@@ -24,23 +24,23 @@ const service = "dataset-api"
 // ErrInvalidDatasetAPIResponse is returned when the dataset api does not respond
 // with a valid status
 type ErrInvalidDatasetAPIResponse struct {
-	actualCode int
+	ActualCode int
```

Comment:
Do we need to export this field? (there is a getter function `Code()` which returns it)

Reply:
I need to set this value to different status codes as observation importer extracts the code to handle the error message it returns within application, so this can be exported or I create a SetCode method